packages/lumberaxe/Gemfile.lock

```diff
@@ -12,99 +12,117 @@ PATH
   remote: .
   specs:
     lumberaxe (0.1.4)
-      activesupport (>= 6.0.6.1, < 7.1)
+      activesupport (= 7.1.3)
```

Comment:
This should be the same as the gemspec, `"activesupport", ">= 6.0.6.1", "<= 7.1"`